src/main/java/address/controller/MainController.java

```diff
@@ -116,29 +117,23 @@ private void showTipOfTheDay() {
     public void initRootLayout() {
         logger.debug("Initializing root layout.");
         final String fxmlResourcePath = FXML_ROOT_LAYOUT;
-        try {
-            // Load root layout from fxml file.
-            FXMLLoader loader = loadFxml(fxmlResourcePath);
-            rootLayout = loader.load();
-
-            // Show the scene containing the root layout.
-            Scene scene = new Scene(rootLayout);
-            scene.setOnKeyPressed(event -> raisePotentialEvent(new KeyBindingEvent(event)));
-            setMinSize();
-            setDefaultSize();
-            primaryStage.setScene(scene);
+        // Load root layout from fxml file.
+        FXMLLoader loader = loadFxml(fxmlResourcePath);
+        rootLayout = (VBox) loadLoader(loader, "Error initializing root layout");
```

Comment:
Looks like FXMLLoader is a temp object. May be abstract both lines into a method?